### PR TITLE
✨ Add Koordlet & Koord-Controller

### PR DESCRIPTION
**Pull request by jasonliu747**
Signed-off-by: Jason Liu <jasonliu747@gmail.com>

### Ⅰ. Describe what this PR does
implement noderesource controller

### CODE REVIEW

File: pkg/common/constants.go [hunk 1]
```diff
@@ -1,9 +1,9 @@
 package common
 
 const (
-	ConfigNameSpace     = "kube-system"
-	SLOConfigMapName    = "slo-manager-config"
-	ColocationConfigKey = "colocation-config"
+	ConfigNameSpace        = "koord-system"
+	KoordCtrlConfigMapName = "koord-controller-config"
```
Comment (hormes) [inline pkg/common/constants.go:5]: 外来只有打包的时候会叫这个名字，内部还是以具体的功能命名？

File: pkg/features/koord_ctrl_features.go [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package features
+
+import (
+	"k8s.io/apimachinery/pkg/util/runtime"
+	"k8s.io/component-base/featuregate"
+)
```
Comment (hormes) [inline pkg/features/koord_ctrl_features.go:22]: rename to slo_controller_features.go ?

File: pkg/common/constants.go (hunk 1 above)
Comment (jasonliu747) [reply pkg/common/constants.go]: SGTM, rename to `slo-controller-config`

File: pkg/features/koord_ctrl_features.go (hunk 2 above)
Comment (jasonliu747) [reply pkg/features/koord_ctrl_features.go]: per our discussion, rename to koord_features

**Review by hormes — APPROVED** [review]

**Pull request merged**